gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -53,18 +93,17 @@ The configuration file used in the following examples demonstrates how to redire
 +
 [source,ssh]
 ----
-sudo touch /usr/local/share/mulesoft/flex-gateway/conf.d/logs-splunk-config.yaml
+sudo touch /usr/local/share/mulesoft/flex-gateway/conf.d/logs-config.yaml
 ----
 
-. Update the file with your Splunk configuration details. For example:
+. Update the file with your desired output configuration details in the following sections:
```

Comment:
is "in the following sections" where they do the updating? Or should this be something like "as shown in the following examples"?